test.py

```diff
@@ -113,7 +130,7 @@ def test_compress_value(self):
 
     def test_compress_input_shape(self):
         '''
-        Test that correct image has correct shape
+        Test that correct input image has correct shape
```

Comment:
Should be "Test that input image has correct shape".

Reply:
Changed this.